docs/conf.py

```diff
@@ -11,7 +11,7 @@
 
 # -- Path setup --------------------------------------------------------------
 
-PROJECT_ROOT_DIR = Path(__file__).parents[1].resolve()
+PROJECT_ROOT_DIR = Path(__file__).resolve().parents[1]
```

Comment:
I don't like trying to maintain these "downstream" — it'd need to be synchronized back. pylint has some configuration for declaring unobvious dynamic attrs. Also, this is likely to be runtime-dependent (it's often the case with pylint).

Oh, and IIRC `__file__` is probably already an absolute path. So is the `resolve()` really needed? Not sure how to react to pylint misbehaving, though. It's the linter bug, not ours — should we be trying to accommodate linter problems?